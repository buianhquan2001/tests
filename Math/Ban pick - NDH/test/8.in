10
993 0.9855
963 0.7157
964 0.6544
970 0.7199
960 0.9227
983 0.9629
979 0.5575
966 0.9767
972 0.5977
977 0.7509
