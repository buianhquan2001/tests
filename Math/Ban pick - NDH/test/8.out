894
506
455
514
733
824
384
843
412
546
